src/components/Header/index.js

```diff
@@ -29,9 +28,21 @@ const AuthenticatedHeaderOption = () => {
     });
   }, []);
 
+  const partnerGroupId = user.data.user.partner_group_id;
+
+  const canSpecifyPartnerGroupId =
+    (rolesList.includes("admin") ||
+      rolesList.includes("partner") ||
+      rolesList.includes("partner_view")) &&
+    user.data.user.partner_group_id;
+
   const canSpecifyUserBaseRole = rolesList.indexOf("admin") > -1;
 
-  const canSpecifyPartner = rolesList.includes("partner") && partnerId != null;
+  const canSpecifyPartner =
+    (rolesList.includes("partner") ||
+      // rolesList.includes("partner_view") ||
+      rolesList.includes("partner_edit")) &&
```

Comment:
After importing the function:
```suggestion
    hasOneFrom(rolesList, ["partner", "partner_edit"]) &&
```